log2ram.conf

```diff
@@ -7,6 +7,15 @@
 # You will need to increase it if you have a server and a lot of log for example.
 SIZE=40M
 
+# Select the log syncing method between the log directory on disk and in the RAM.
+# The variable may be uncommented out, setting it to false, if "cp" is preferred over "rsync".
+# Pre 1.6.1 the config default was to use "cp", unless USE_RSYNC was explicitly set to true.
+# Post 1.6.1 the parameter was briefly removed and "rsync" became the default whenever
```

Comment:
We don't need pre/post 1.6.1 explanation. on the config.
I will add that to the release note.

Reply:
Thank you.